api/envoy/extensions/filters/http/dynamic_forward_proxy/v3/dynamic_forward_proxy.proto

```diff
@@ -26,14 +28,16 @@ message FilterConfig {
   // The DNS cache configuration that the filter will attach to. Note this configuration must
   // match that of associated :ref:`dynamic forward proxy cluster configuration
   // <envoy_v3_api_field_extensions.clusters.dynamic_forward_proxy.v3.ClusterConfig.dns_cache_config>`.
-  common.dynamic_forward_proxy.v3.DnsCacheConfig dns_cache_config = 1
-      [(validate.rules).message = {required: true}];
+  common.dynamic_forward_proxy.v3.DnsCacheConfig dns_cache_config = 1;
 
   // When this flag is set, the filter will add the resolved upstream address in the filter
   // state. The state should be saved with key
   // ``envoy.stream.upstream_address`` (See
   // :repo:`upstream_address.h<source/common/stream_info/upstream_address.h>`).
   bool save_upstream_address = 2;
+
+  // The timeout used for sub cluster initialization. Defaults to 5s if not set.
+  google.protobuf.Duration cluster_init_timeout = 3 [(validate.rules).duration = {gt {}}];
```

Comment:
Can't this be governed by existing timeouts?

Reply:
Ping on this.

Reply:
oh, sorry. it's the same reason as the above`max_sub_clusters`.
I think we'd better not let the users to mix use `cluster_init_timeout` and `dns_cache_config`.

`dns_cache_config` is for the old implementation, while `cluster_init_timeout` is for the new implementation.